docs/guides/BUILDING.md

```diff
@@ -81,9 +81,8 @@ dependency.
 
 ### Installing prerequisites on Raspberry Pi 4
 
-Using `rpi-imager`, install the Ubuntu _21.04_ 64-bit _server_ OS for arm64
-architectures on a micro SD card. This release will have bluez 5.55 or newer
-which is required for BLE functionality.
+Using `rpi-imager`, install the Ubuntu _22.04_ 64-bit _server_ OS for arm64
+architectures on a micro SD card.
```

Comment:
It's not clear if the "Raspberry Pi specific dependencies" and configuration changes are still needed. I was able to successfully provision a Realtek Ameba board running the Lighting app example over BLE without the steps below.

Happy to test other scenarios to see if they are still needed.